ansible/pmm/main.yml

```diff
@@ -23,7 +23,7 @@
       when: ansible_os_family == 'RedHat'
       yum: name={{ item }} state=installed
       with_items:
-        - screen
```

Comment:
screen is almost useless inside the container; lsof is often needed for debugging open ports